cmake/math/MathLibs.cmake

```diff
@@ -170,10 +170,17 @@ else()
     set(_end_group)
 endif()
 
+# LAB Jul 2017: Because gcc thinks that if OpenMP is on, then implicitly one _must_ want their
+#   libgomp, when we'd actually prefer libiomp5 that doesn't give nthread-dependent energies.
```

Comment:
happily, attending a conference means lots of time to contemplate what on earth just happened and dream of solutions. the I-am-linking-to-pristine-intel-mkl_rt-yet-managed-to-completely-suppress-all-threading bug of the week before was much scarier (soln: any statically-linked mkl can suppress mkl_rt threading).